frontend/src/metabase/query_builder/components/view/QuestionSummaries.jsx

```diff
@@ -1,33 +1,10 @@
 /* eslint-disable react/prop-types */
 import { t } from "ttag";
 
-import PopoverWithTrigger from "metabase/components/PopoverWithTrigger";
-
 import { color } from "metabase/lib/colors";
-import ViewPill from "./ViewPill";
 import ViewButton from "./ViewButton";
 import { HeaderButton } from "./ViewHeader.styled";
 
-import SummarizeSidebar from "./sidebars/SummarizeSidebar/SummarizeSidebar";
-
-const SummarizePill = props => (
-  <ViewPill icon="insight" color={color("summarize")} {...props} />
-);
-export default function QuestionSummaries({
```

Comment:
This component is introduced in https://github.com/metabase/metabase/pull/9592, but I couldn't even find where it's being used.

The only component I found being used in this file is `QuestionSummarizeWidget`.

_the following diff is from https://github.com/metabase/metabase/pull/9592_

https://github.com/metabase/metabase/blob/d3ff5fdc5fca87d3f7e01a39c8890d599a927260/frontend/src/metabase/query_builder/components/view/ViewHeader.jsx#L21